iehfc_app/server.R

```diff
@@ -1,20 +1,22 @@
 pacman::p_load(
     shiny, dplyr, tidyr, stringr, lubridate, purrr, ggplot2, janitor, data.table, DT, remotes, bsicons,
-    shinydashboard, shinyjs, markdown, htmlwidgets, webshot, plotly, bslib, kableExtra, here, bit64, rsconnect, shinycssloaders, httr,jsonlite
+    shinydashboard, shinyjs, markdown, htmlwidgets, webshot, plotly, bslib, kableExtra, here, bit64, rsconnect, shinycssloaders, httr, jsonlite, pysparklyr,reticulate,sparklyr,dbplyr
 )
 
   # Increase maximum file upload size
   
   options(shiny.maxRequestSize = 100 * (1024^2))
-
+  
   iehfc_server <- function(input, output, session) {
       
-      source(file.path(getwd(),  "server_scripts", "duplicates.R"), local = TRUE)
-      source(file.path(getwd(),  "server_scripts", "outliers.R"), local = TRUE)
-      source(file.path(getwd(),  "server_scripts", "enumerator.R"), local = TRUE)
-      source(file.path(getwd(),  "server_scripts", "admin.R"), local = TRUE)
-      source(file.path(getwd(),  "server_scripts", "unit.R"), local = TRUE)
-      source(file.path(getwd(),  "server_scripts", "summary.R"), local = TRUE)
+    source("server_scripts/duplicates.R", local = TRUE)
+    source("server_scripts/outliers.R", local = TRUE)
+    source("server_scripts/enumerator.R", local = TRUE)
+    source("server_scripts/admin.R", local = TRUE)
+    source("server_scripts/unit.R", local = TRUE)
+    source("server_scripts/summary.R", local = TRUE)
+    source("server_scripts/connection.R", local = TRUE)
```

Comment:
@yukinko-iwasaki did you test that this change works with windows? The original implementation might be more platform agnostic